.gitignore

```diff
@@ -25,10 +25,11 @@ yarn-debug.log*
 yarn-error.log*
 
 # local env files
+.env
```

Comment:
Note for future: Is there a sample `.env` anywhere? I do not see one.

Reply:
we have a shared 1Password vault. let's figure out how to get you the tokens